### PR TITLE
Trying to improve Travis build stability

### PR DESCRIPTION
**Pull request by sfriedowitz**
The change here causes our test runner to log each test it runs, which I am hoping will alleviate the travis timeout issue.

There is also one test that had about a 5% failure rate. I changed a param to alleviate that.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -34,6 +34,8 @@ ThisBuild / libraryDependencies ++= {
   }
 }
 
+ThisBuild / Test / testOptions += Tests.Argument(TestFrameworks.JUnit, "-v")
```
Comment (sfriedowitz) [inline build.sbt:37]: This causes the `JUnit` test runner to log the test start/complete notifications to stdout, so we get console output of the test suite status as it runs.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -14,7 +14,7 @@ object Dependencies {
     "org.scalanlp"    %% "breeze"          % breezeVersion,
     "junit"            % "junit"           % junitVersion     % "test",
     "org.scalatest"   %% "scalatest"       % scalaTestVersion % "test",
-    "com.novocode"     % "junit-interface" % "0.11"           % "test",
+    "com.github.sbt"   % "junit-interface" % "0.13.3"         % "test",
```
Comment (sfriedowitz) [inline project/Dependencies.scala:17]: They changed their name and updated.

File: src/test/scala/io/citrine/lolo/trees/regression/RegressionTreeTest.scala [hunk 3]
```diff
@@ -244,21 +244,20 @@ class RegressionTreeTest {
     */
   @Test
   def testSimpleBoltzmannTree(): Unit = {
```
Comment (sfriedowitz) [inline src/test/scala/io/citrine/lolo/trees/regression/RegressionTreeTest.scala:246]: This test flaked for me a few times.

File: .travis.yml [hunk 4]
```diff
@@ -24,25 +24,25 @@ jobs:
   include:
   - stage: test
     script:
-    - sbt scalafmtCheckAll
-    - sbt +test
+      - sbt -Dsbt.log.noformat=true scalafmtCheckAll
```
Comment (sfriedowitz) [inline .travis.yml:27]: This removes extra formatting from the travis logs.

**Review by bfolie — APPROVED** [review]

**Pull request merged**